Dependabot fails to submit PRs for NuGet updates when lockfiles are used

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Package ecosystem

NuGet

### Package manager version

_No response_

### Language version

_No response_

### Manifest location and content before the Dependabot update

`src/Directory.Packages.props`:

```xml
<Project>
  <PropertyGroup>
    <ManagePackageVersionsCentrally>true</ManagePackageVersionsCentrally>
    <CentralPackageTransitivePinningEnabled>true</CentralPackageTransitivePinningEnabled>
  </PropertyGroup>
  <ItemGroup>
    <PackageVersion Include="Scrutor" Version="5.0.0" />
    <PackageVersion Include="xunit" Version="2.9.1" />
  </ItemGroup>
</Project>
```

`src/Directory.Build.targets`:
```xml
<Project>
    <PropertyGroup>
        <Nullable>enable</Nullable>
        <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
    </PropertyGroup>

    <!-- GitHub Actions NuGet cache recommendation -->
    <PropertyGroup>
        <DisableImplicitNuGetFallbackFolder>true</DisableImplicitNuGetFallbackFolder>
        <RestorePackagesWithLockFile>true</RestorePackagesWithLockFile>
    </PropertyGroup>
</Project>
```

`src/TestProject.csproj`:
```xml
<Project Sdk="Microsoft.NET.Sdk.Web">
  <PropertyGroup>
    <TargetFramework>net8.0</TargetFramework>
  </PropertyGroup>
  <ItemGroup>
    <PackageReference Include="Scrutor" />
  </ItemGroup>
</Project>
```

`src/packages.lock.json`:
```json
{
  "version": 2,
  "dependencies": {
    "net8.0": {
      "Scrutor": {
        "type": "Direct",
        "requested": "[5.0.0, )",
        "resolved": "5.0.0",
        "contentHash": "JrqAg2HuvUUZQP0G4tM0hOFU0yr8EcZJRyl5nNQJJxvll9+UssRaySRzybkZYC2AcNwVoLfg5lyQum2+5pSVDw==",
        "dependencies": {
          "Microsoft.Extensions.DependencyInjection.Abstractions": "8.0.1",
          "Microsoft.Extensions.DependencyModel": "8.0.1"
        }
      },
      "Microsoft.Extensions.DependencyInjection.Abstractions": {
        "type": "Transitive",
        "resolved": "8.0.1",
        "contentHash": "fGLiCRLMYd00JYpClraLjJTNKLmMJPnqxMaiRzEBIIvevlzxz33mXy39Lkd48hu1G+N21S7QpaO5ZzKsI6FRuA=="
      },
      "Microsoft.Extensions.DependencyModel": {
        "type": "Transitive",
        "resolved": "8.0.1",
        "contentHash": "5Ou6varcxLBzQ+Agfm0k0pnH7vrEITYlXMDuE6s7ZHlZHz6/G8XJ3iISZDr5rfwfge6RnXJ1+Wc479mMn52vjA==",
        "dependencies": {
          "System.Text.Encodings.Web": "8.0.0",
          "System.Text.Json": "8.0.4"
        }
      },
      "System.Text.Encodings.Web": {
        "type": "Transitive",
        "resolved": "8.0.0",
        "contentHash": "yev/k9GHAEGx2Rg3/tU6MQh4HGBXJs70y7j1LaM1i/ER9po+6nnQ6RRqTJn1E7Xu0fbIFK80Nh5EoODxrbxwBQ=="
      },
      "System.Text.Json": {
        "type": "Transitive",
        "resolved": "8.0.4",
        "contentHash": "bAkhgDJ88XTsqczoxEMliSrpijKZHhbJQldhAmObj/RbrN3sU5dcokuXmWJWsdQAhiMJ9bTayWsL1C9fbbCRhw==",
        "dependencies": {
          "System.Text.Encodings.Web": "8.0.0"
        }
      }
    }
  }
}
```

### dependabot.yml content

version: 2
updates:
  - package-ecosystem: "nuget"
    directory: "/"
    schedule:
      interval: "weekly"

### Updated dependency

Scrutor - 5.0.0 to 5.0.1
xunit 2.9.1 to 2.9.2

Those 2 are in my repro, but it fails the same for all dependencies.

### What you expected to see, versus what you actually saw

I expected to see Scrutor and xunit bumped in the `src/Directory.Packages.props` file and the `src/packages.lock.json` to be regenerated accordingly. Until 25th of September, Dependabot would create PRs with just the `Directory.Packages.props` update, without the `packages.lock.json` - those would then fail to build, but I could update them manually and merge. After 25th of September, no PRs are created at all.

There are two different bugs, I think:
- for Scrutor, an update is being generated but fails to be submitted
- for xunit (which is not linked to in `.csproj`), Dependabot fails to even generate an update to `Directory.Packages.props`

```
Dependabot encountered '2' error(s) during execution, please check the logs for more details.
+-------------------------------+
| Dependencies failed to update |
+---------+---------------------+
| Scrutor | unknown_error       |
| xunit   | update_not_possible |
+---------+---------------------+
```

Note: this is GitHub Enterprise (but on github.com).

Full logs follow.

### Native package manager behavior

_No response_

### Images of the diff or a link to the PR, issue, or logs

<details><summary>Full failed update logfile</summary>
```
2024-10-04T13:13:57.1369432Z Current runner version: '2.319.1'
2024-10-04T13:13:57.1395333Z ##[group]Operating System
2024-10-04T13:13:57.1396116Z Ubuntu
2024-10-04T13:13:57.1396487Z 22.04.5
2024-10-04T13:13:57.1396825Z LTS
2024-10-04T13:13:57.1397256Z ##[endgroup]
2024-10-04T13:13:57.1397650Z ##[group]Runner Image
2024-10-04T13:13:57.1398167Z Image: ubuntu-22.04
2024-10-04T13:13:57.1398545Z Version: 20240922.1.0
2024-10-04T13:13:57.1399580Z Included Software: https://github.com/actions/runner-images/blob/ubuntu22/20240922.1/images/ubuntu/Ubuntu2204-Readme.md
2024-10-04T13:13:57.1401135Z Image Release: https://github.com/actions/runner-images/releases/tag/ubuntu22%2F20240922.1
2024-10-04T13:13:57.1402021Z ##[endgroup]
2024-10-04T13:13:57.1402431Z ##[group]Runner Image Provisioner
2024-10-04T13:13:57.1402993Z 2.0.384.1
2024-10-04T13:13:57.1403319Z ##[endgroup]
2024-10-04T13:13:57.1419631Z ##[group]GITHUB_TOKEN Permissions
2024-10-04T13:13:57.1421322Z Contents: read
2024-10-04T13:13:57.1421873Z Metadata: read
2024-10-04T13:13:57.1422484Z Packages: read
2024-10-04T13:13:57.1423035Z ##[endgroup]
2024-10-04T13:13:57.1426457Z Secret source: None
2024-10-04T13:13:57.1427175Z Prepare workflow directory
2024-10-04T13:13:57.2084595Z Prepare all required actions
2024-10-04T13:13:57.2255282Z Getting action download info
2024-10-04T13:13:57.3678084Z Download action repository 'github/dependabot-action@main' (SHA:b293965467821a9fe9ed370cac09d681c7a4ed6c)
2024-10-04T13:13:57.9307979Z Complete job name: Dependabot
2024-10-04T13:13:58.0417364Z ##[group]Run mkdir -p  ./dependabot-job-895966952-1728047626
2024-10-04T13:13:58.0418487Z [36;1mmkdir -p  ./dependabot-job-895966952-1728047626[0m
2024-10-04T13:13:58.0447692Z shell: /usr/bin/bash -e {0}
2024-10-04T13:13:58.0448239Z ##[endgroup]
2024-10-04T13:13:58.1222397Z ##[group]Run github/dependabot-action@main
2024-10-04T13:13:58.1223650Z env:
2024-10-04T13:13:58.1224871Z   DEPENDABOT_DISABLE_CLEANUP: 1
2024-10-04T13:13:58.1226293Z   DEPENDABOT_ENABLE_CONNECTIVITY_CHECK: 0
2024-10-04T13:13:58.1227863Z   GITHUB_TOKEN: ***
2024-10-04T13:13:58.1229617Z   GITHUB_DEPENDABOT_JOB_TOKEN: ***
2024-10-04T13:13:58.1231410Z   GITHUB_DEPENDABOT_CRED_TOKEN: ***
2024-10-04T13:13:58.1232339Z ##[endgroup]
2024-10-04T13:13:58.3993143Z 🤖 ~ starting update ~
2024-10-04T13:13:58.4040804Z Fetching job details
2024-10-04T13:13:58.7702442Z ##[group]Pulling updater images
2024-10-04T13:13:58.7714739Z Pulling image ghcr.io/dependabot/dependabot-updater-nuget:c557bd79ea8c42822834017211f640815947eaed...
2024-10-04T13:14:41.8513294Z Pulled image ghcr.io/dependabot/dependabot-updater-nuget:c557bd79ea8c42822834017211f640815947eaed
2024-10-04T13:14:41.8535851Z Pulling image ghcr.io/github/dependabot-update-job-proxy/dependabot-update-job-proxy:v2.0.20240919194919@sha256:a8861ab9eba169d6c8071b3e2d8d67e7a23b87a6566af87255b746e2e081bca3...
2024-10-04T13:14:42.4152088Z Pulled image ghcr.io/github/dependabot-update-job-proxy/dependabot-update-job-proxy:v2.0.20240919194919@sha256:a8861ab9eba169d6c8071b3e2d8d67e7a23b87a6566af87255b746e2e081bca3
2024-10-04T13:14:42.4159022Z ##[endgroup]
2024-10-04T13:14:42.4159701Z Starting update process
2024-10-04T13:14:42.7392074Z Created proxy container: c6ff44976a9513168aa53dc2bfe13d8b7a5e45a67a78a4956c92a6aab178647a
2024-10-04T13:14:43.0521620Z Created container: d3fcc3070906824ff3832801c49309ab362c4fb1ed649365b578c307611c1c6c
2024-10-04T13:14:43.0917713Z   proxy | 2024/10/04 13:14:43 proxy starting, commit: cf3f47452d431f45a8cdd6282ca0c7682bafa86d
2024-10-04T13:14:43.0951692Z   proxy | 2024/10/04 13:14:43 Listening (:1080)
2024-10-04T13:14:43.2668383Z updater | Updating certificates in /etc/ssl/certs...
2024-10-04T13:14:44.1849568Z updater | rehash: warning: skipping ca-certificates.crt,it does not contain exactly one certificate or CRL
2024-10-04T13:14:44.2362493Z updater | 1 added, 0 removed; done.
2024-10-04T13:14:44.2363708Z Running hooks in /etc/ca-certificates/update.d...
2024-10-04T13:14:44.2416838Z updater | done.
2024-10-04T13:14:44.2945059Z updater | NuGet native updater experiment value: null
2024-10-04T13:14:47.0146231Z updater | 2024/10/04 13:14:47 INFO <job_895966952> Starting job processing
2024-10-04T13:14:47.0159370Z 2024/10/04 13:14:47 INFO <job_895966952> Job definition: {"job":{"allowed-updates":[{"dependency-type":"direct","update-type":"all"}],"commit-message-options":{"prefix":null,"prefix-development":null,"include-scope":null},"credentials-metadata":[{"type":"git_source","host":"github.com"}],"debug":null,"dependencies":null,"dependency-groups":[],"dependency-group-to-refresh":null,"existing-pull-requests":[],"existing-group-pull-requests":[],"experiments":{"record-ecosystem-versions":true,"record-update-job-unknown-error":true,"proxy-cached":true,"move-job-token":true,"dependency-change-validation":true,"add-deprecation-warn-to-pr-message":true},"ignore-conditions":[],"lockfile-only":false,"max-updater-run-time":2700,"package-manager":"nuget","proxy-log-response-body-on-auth-failure":true,"requirements-update-strategy":null,"reject-external-code":false,"security-advisories":[],"security-updates-only":false,"source":{"provider":"github","repo":"name-redacted/dependabot-bug-repro","branch":null,"directory":"/.","api-endpoint":"https://api.github.com/","hostname":"github.com"},"updating-a-pull-request":false,"update-subdependencies":false,"vendor-dependencies":false,"repo-private":true}}
2024-10-04T13:14:47.2141849Z   proxy | 2024/10/04 13:14:47 [002] GET https://github.com:443/name-redacted/dependabot-bug-repro/info/refs?service=git-upload-pack
2024-10-04T13:14:47.2157543Z   proxy | 2024/10/04 13:14:47 [002] * authenticating git server request (host: github.com)
2024-10-04T13:14:47.3111657Z   proxy | 2024/10/04 13:14:47 [002] 200 https://github.com:443/name-redacted/dependabot-bug-repro/info/refs?service=git-upload-pack
2024-10-04T13:14:47.3443895Z   proxy | 2024/10/04 13:14:47 [004] POST https://github.com:443/name-redacted/dependabot-bug-repro/git-upload-pack
2024-10-04T13:14:47.3445338Z 2024/10/04 13:14:47 [004] * authenticating git server request (host: github.com)
2024-10-04T13:14:47.4215160Z   proxy | 2024/10/04 13:14:47 [004] 200 https://github.com:443/name-redacted/dependabot-bug-repro/git-upload-pack
2024-10-04T13:14:47.4557857Z   proxy | 2024/10/04 13:14:47 [006] POST https://github.com:443/name-redacted/dependabot-bug-repro/git-upload-pack
2024-10-04T13:14:47.4560873Z 2024/10/04 13:14:47 [006] * authenticating git server request (host: github.com)
2024-10-04T13:14:47.5477172Z   proxy | 2024/10/04 13:14:47 [006] 200 https://github.com:443/name-redacted/dependabot-bug-repro/git-upload-pack
2024-10-04T13:14:47.7143867Z updater | 2024/10/04 13:14:47 INFO <job_895966952> Base commit SHA: a778134c5a29f99a585f1d22c51f7d5c823d49ca
2024-10-04T13:14:47.7181170Z updater | 2024/10/04 13:14:47 INFO <job_895966952> Finished job processing
2024-10-04T13:14:47.7597739Z updater | NuGet native updater experiment value: null
2024-10-04T13:14:50.3535964Z updater | 2024/10/04 13:14:50 INFO <job_895966952> Starting job processing
2024-10-04T13:14:50.4150200Z updater | running NuGet discovery:
2024-10-04T13:14:50.4152963Z /opt/nuget/NuGetUpdater/NuGetUpdater.Cli discover --repo-root /home/dependabot/dependabot-updater/repo --workspace / --output /tmp/.dependabot/discovery.1.json --verbose
2024-10-04T13:14:52.5173368Z   proxy | 2024/10/04 13:14:52 [008] GET https://api.nuget.org:443/v3/index.json
[-------redacted for brevity ------------------ ]
2024-10-04T13:14:54.0368303Z updater | Discovering build files in workspace [/home/dependabot/dependabot-updater/repo].
2024-10-04T13:14:54.0369695Z   No dotnet-tools.json file found.
2024-10-04T13:14:54.0370455Z   No global.json file found.
2024-10-04T13:14:54.0371200Z   Discovering projects beneath [.].
2024-10-04T13:14:54.0372065Z   No packages.config file found.
2024-10-04T13:14:54.0372952Z   Discovered [src/Directory.Packages.props] file.
2024-10-04T13:14:54.0396516Z Discovery complete.
2024-10-04T13:14:54.0397513Z updater | 2024/10/04 13:14:54 INFO <job_895966952> Discovery JSON content: {
2024-10-04T13:14:54.0398565Z   "Path": "",
2024-10-04T13:14:54.0399109Z   "IsSuccess": true,
2024-10-04T13:14:54.0400050Z   "Projects": [
2024-10-04T13:14:54.0400612Z     {
2024-10-04T13:14:54.0401255Z       "FilePath": "src/Directory.Build.targets",
2024-10-04T13:14:54.0402136Z       "Dependencies": [],
2024-10-04T13:14:54.0402752Z       "IsSuccess": true,
2024-10-04T13:14:54.0403415Z       "Properties": [],
2024-10-04T13:14:54.0404451Z       "TargetFrameworks": [],
2024-10-04T13:14:54.0405221Z       "ReferencedProjectPaths": []
2024-10-04T13:14:54.0405909Z     },
2024-10-04T13:14:54.0406500Z     {
2024-10-04T13:14:54.0407082Z       "FilePath": "src/TestProject.csproj",
2024-10-04T13:14:54.0407939Z       "Dependencies": [
2024-10-04T13:14:54.0408581Z         {
2024-10-04T13:14:54.0409354Z           "Name": "Microsoft.Extensions.DependencyInjection.Abstractions",
2024-10-04T13:14:54.0410427Z           "Version": "8.0.1",
2024-10-04T13:14:54.0411113Z           "Type": "Unknown",
2024-10-04T13:14:54.0411785Z           "EvaluationResult": null,
2024-10-04T13:14:54.0412610Z           "TargetFrameworks": [
2024-10-04T13:14:54.0413302Z             "net8.0"
2024-10-04T13:14:54.0413863Z           ],
2024-10-04T13:14:54.0414865Z           "IsDevDependency": false,
2024-10-04T13:14:54.0415543Z           "IsDirect": false,
2024-10-04T13:14:54.0416225Z           "IsTransitive": true,
2024-10-04T13:14:54.0416992Z           "IsOverride": false,
2024-10-04T13:14:54.0417568Z           "IsUpdate": false,
2024-10-04T13:14:54.0418206Z           "InfoUrl": null
2024-10-04T13:14:54.0418724Z         },
2024-10-04T13:14:54.0419164Z         {
2024-10-04T13:14:54.0419885Z           "Name": "Microsoft.Extensions.DependencyModel",
2024-10-04T13:14:54.0420753Z           "Version": "8.0.1",
2024-10-04T13:14:54.0421697Z           "Type": "Unknown",
2024-10-04T13:14:54.0422425Z           "EvaluationResult": null,
2024-10-04T13:14:54.0423146Z           "TargetFrameworks": [
2024-10-04T13:14:54.0423842Z             "net8.0"
2024-10-04T13:14:54.0424902Z           ],
2024-10-04T13:14:54.0425514Z           "IsDevDependency": false,
2024-10-04T13:14:54.0426320Z           "IsDirect": false,
2024-10-04T13:14:54.0426968Z           "IsTransitive": true,
2024-10-04T13:14:54.0427671Z           "IsOverride": false,
2024-10-04T13:14:54.0428356Z           "IsUpdate": false,
2024-10-04T13:14:54.0429028Z           "InfoUrl": null
2024-10-04T13:14:54.0429640Z         },
2024-10-04T13:14:54.0430217Z         {
2024-10-04T13:14:54.0430823Z           "Name": "Microsoft.NET.Sdk.Web",
2024-10-04T13:14:54.0431582Z           "Version": null,
2024-10-04T13:14:54.0432330Z           "Type": "MSBuildSdk",
2024-10-04T13:14:54.0433046Z           "EvaluationResult": null,
2024-10-04T13:14:54.0433844Z           "TargetFrameworks": null,
2024-10-04T13:14:54.0434904Z           "IsDevDependency": false,
2024-10-04T13:14:54.0435630Z           "IsDirect": false,
2024-10-04T13:14:54.0436461Z           "IsTransitive": false,
2024-10-04T13:14:54.0437198Z           "IsOverride": false,
2024-10-04T13:14:54.0437863Z           "IsUpdate": false,
2024-10-04T13:14:54.0438612Z           "InfoUrl": null
2024-10-04T13:14:54.0439221Z         },
2024-10-04T13:14:54.0439730Z         {
2024-10-04T13:14:54.0440297Z           "Name": "Scrutor",
2024-10-04T13:14:54.0440951Z           "Version": "5.0.0",
2024-10-04T13:14:54.0441633Z           "Type": "PackageReference",
2024-10-04T13:14:54.0442465Z           "EvaluationResult": {
2024-10-04T13:14:54.0443171Z             "ResultType": "Success",
2024-10-04T13:14:54.0444360Z             "OriginalValue": "5.0.0",
2024-10-04T13:14:54.0445154Z             "EvaluatedValue": "5.0.0",
2024-10-04T13:14:54.0445915Z             "RootPropertyName": null,
2024-10-04T13:14:54.0446758Z             "ErrorMessage": null
2024-10-04T13:14:54.0447363Z           },
2024-10-04T13:14:54.0447961Z           "TargetFrameworks": [
2024-10-04T13:14:54.0448691Z             "net8.0"
2024-10-04T13:14:54.0449252Z           ],
2024-10-04T13:14:54.0449839Z           "IsDevDependency": false,
2024-10-04T13:14:54.0450609Z           "IsDirect": true,
2024-10-04T13:14:54.0451532Z           "IsTransitive": false,
2024-10-04T13:14:54.0452292Z           "IsOverride": false,
2024-10-04T13:14:54.0452914Z           "IsUpdate": false,
2024-10-04T13:14:54.0453585Z           "InfoUrl": null
2024-10-04T13:14:54.0454620Z         },
2024-10-04T13:14:54.0455175Z         {
2024-10-04T13:14:54.0455746Z           "Name": "System.Text.Json",
2024-10-04T13:14:54.0456550Z           "Version": "8.0.4",
2024-10-04T13:14:54.0457240Z           "Type": "Unknown",
2024-10-04T13:14:54.0457910Z           "EvaluationResult": null,
2024-10-04T13:14:54.0458748Z           "TargetFrameworks": [
2024-10-04T13:14:54.0459365Z             "net8.0"
2024-10-04T13:14:54.0459926Z           ],
2024-10-04T13:14:54.0460607Z           "IsDevDependency": false,
2024-10-04T13:14:54.0461317Z           "IsDirect": false,
2024-10-04T13:14:54.0462048Z           "IsTransitive": true,
2024-10-04T13:14:54.0462776Z           "IsOverride": false,
2024-10-04T13:14:54.0463434Z           "IsUpdate": false,
2024-10-04T13:14:54.0464566Z           "InfoUrl": null
2024-10-04T13:14:54.0465201Z         }
2024-10-04T13:14:54.0465712Z       ],
2024-10-04T13:14:54.0466285Z       "IsSuccess": true,
2024-10-04T13:14:54.0466917Z       "Properties": [
2024-10-04T13:14:54.0467509Z         {
2024-10-04T13:14:54.0468307Z           "Name": "CentralPackageTransitivePinningEnabled",
2024-10-04T13:14:54.0469188Z           "Value": "true",
2024-10-04T13:14:54.0470044Z           "SourceFilePath": "src/Directory.Packages.props"
2024-10-04T13:14:54.0470880Z         },
2024-10-04T13:14:54.0471395Z         {
2024-10-04T13:14:54.0472141Z           "Name": "DisableImplicitNuGetFallbackFolder",
2024-10-04T13:14:54.0472950Z           "Value": "true",
2024-10-04T13:14:54.0474186Z           "SourceFilePath": "src/Directory.Build.targets"
2024-10-04T13:14:54.0475239Z         },
2024-10-04T13:14:54.0475757Z         {
2024-10-04T13:14:54.0476418Z           "Name": "ManagePackageVersionsCentrally",
2024-10-04T13:14:54.0477337Z           "Value": "true",
2024-10-04T13:14:54.0478165Z           "SourceFilePath": "src/Directory.Packages.props"
2024-10-04T13:14:54.0479077Z         },
2024-10-04T13:14:54.0479610Z         {
2024-10-04T13:14:54.0480145Z           "Name": "Nullable",
2024-10-04T13:14:54.0480878Z           "Value": "enable",
2024-10-04T13:14:54.0481689Z           "SourceFilePath": "src/Directory.Build.targets"
2024-10-04T13:14:54.0482530Z         },
2024-10-04T13:14:54.0483117Z         {
2024-10-04T13:14:54.0483759Z           "Name": "RestorePackagesWithLockFile",
2024-10-04T13:14:54.0485001Z           "Value": "true",
2024-10-04T13:14:54.0485875Z           "SourceFilePath": "src/Directory.Build.targets"
2024-10-04T13:14:54.0486657Z         },
2024-10-04T13:14:54.0487167Z         {
2024-10-04T13:14:54.0487844Z           "Name": "TargetFramework",
2024-10-04T13:14:54.0488596Z           "Value": "net8.0",
2024-10-04T13:14:54.0489435Z           "SourceFilePath": "src/TestProject.csproj"
2024-10-04T13:14:54.0490262Z         },
2024-10-04T13:14:54.0490796Z         {
2024-10-04T13:14:54.0491516Z           "Name": "TreatWarningsAsErrors",
2024-10-04T13:14:54.0492303Z           "Value": "true",
2024-10-04T13:14:54.0493017Z           "SourceFilePath": "src/Directory.Build.targets"
2024-10-04T13:14:54.0493934Z         }
2024-10-04T13:14:54.0494658Z       ],
2024-10-04T13:14:54.0495292Z       "TargetFrameworks": [
2024-10-04T13:14:54.0496025Z         "net8.0"
2024-10-04T13:14:54.0496559Z       ],
2024-10-04T13:14:54.0497206Z       "ReferencedProjectPaths": []
2024-10-04T13:14:54.0497899Z     }
2024-10-04T13:14:54.0498333Z   ],
2024-10-04T13:14:54.0498988Z   "DirectoryPackagesProps": {
2024-10-04T13:14:54.0499788Z     "FilePath": "src/Directory.Packages.props",
2024-10-04T13:14:54.0500659Z     "IsSuccess": true,
2024-10-04T13:14:54.0501404Z     "IsTransitivePinningEnabled": false,
2024-10-04T13:14:54.0502175Z     "Dependencies": [
2024-10-04T13:14:54.0502809Z       {
2024-10-04T13:14:54.0503478Z         "Name": "Scrutor",
2024-10-04T13:14:54.0504476Z         "Version": "5.0.0",
2024-10-04T13:14:54.0505452Z         "Type": "PackageVersion",
2024-10-04T13:14:54.0506254Z         "EvaluationResult": {
2024-10-04T13:14:54.0506962Z           "ResultType": "Success",
2024-10-04T13:14:54.0507737Z           "OriginalValue": "5.0.0",
2024-10-04T13:14:54.0508487Z           "EvaluatedValue": "5.0.0",
2024-10-04T13:14:54.0509233Z           "RootPropertyName": null,
2024-10-04T13:14:54.0510021Z           "ErrorMessage": null
2024-10-04T13:14:54.0510688Z         },
2024-10-04T13:14:54.0511194Z         "TargetFrameworks": null,
2024-10-04T13:14:54.0511967Z         "IsDevDependency": false,
2024-10-04T13:14:54.0512683Z         "IsDirect": true,
2024-10-04T13:14:54.0513319Z         "IsTransitive": false,
2024-10-04T13:14:54.0514316Z         "IsOverride": false,
2024-10-04T13:14:54.0515088Z         "IsUpdate": false,
2024-10-04T13:14:54.0515802Z         "InfoUrl": null
2024-10-04T13:14:54.0516426Z       },
2024-10-04T13:14:54.0516881Z       {
2024-10-04T13:14:54.0517496Z         "Name": "xunit",
2024-10-04T13:14:54.0518175Z         "Version": "2.9.1",
2024-10-04T13:14:54.0518835Z         "Type": "PackageVersion",
2024-10-04T13:14:54.0519651Z         "EvaluationResult": {
2024-10-04T13:14:54.0520350Z           "ResultType": "Success",
2024-10-04T13:14:54.0521070Z           "OriginalValue": "2.9.1",
2024-10-04T13:14:54.0521911Z           "EvaluatedValue": "2.9.1",
2024-10-04T13:14:54.0522637Z           "RootPropertyName": null,
2024-10-04T13:14:54.0523415Z           "ErrorMessage": null
2024-10-04T13:14:54.0524590Z         },
2024-10-04T13:14:54.0525192Z         "TargetFrameworks": null,
2024-10-04T13:14:54.0526024Z         "IsDevDependency": false,
2024-10-04T13:14:54.0526725Z         "IsDirect": true,
2024-10-04T13:14:54.0527595Z         "IsTransitive": false,
2024-10-04T13:14:54.0528364Z         "IsOverride": false,
2024-10-04T13:14:54.0529029Z         "IsUpdate": false,
2024-10-04T13:14:54.0529649Z         "InfoUrl": null
2024-10-04T13:14:54.0530383Z       }
2024-10-04T13:14:54.0530905Z     ]
2024-10-04T13:14:54.0531413Z   },
2024-10-04T13:14:54.0531994Z   "GlobalJson": null,
2024-10-04T13:14:54.0532643Z   "DotNetToolsJson": null,
2024-10-04T13:14:54.0533376Z   "ErrorType": null,
2024-10-04T13:14:54.0534232Z   "ErrorDetails": null
2024-10-04T13:14:54.0534868Z }
2024-10-04T13:14:54.3237026Z   proxy | 2024/10/04 13:14:54 [034] POST /update_jobs/895966952/update_dependency_list
2024-10-04T13:14:54.3763810Z   proxy | 2024/10/04 13:14:54 [034] 204 /update_jobs/895966952/update_dependency_list
2024-10-04T13:14:54.4253118Z   proxy | 2024/10/04 13:14:54 [036] POST /update_jobs/895966952/increment_metric
2024-10-04T13:14:54.4529495Z   proxy | 2024/10/04 13:14:54 [036] 204 /update_jobs/895966952/increment_metric
2024-10-04T13:14:54.4569608Z updater | 2024/10/04 13:14:54 INFO <job_895966952> Starting update job for name-redacted/dependabot-bug-repro
2024-10-04T13:14:54.4585384Z 2024/10/04 13:14:54 INFO <job_895966952> Checking all dependencies for version updates...
2024-10-04T13:14:54.4616226Z updater | 2024/10/04 13:14:54 INFO <job_895966952> Checking if Scrutor 5.0.0 needs updating
2024-10-04T13:14:54.5674128Z   proxy | 2024/10/04 13:14:54 [038] GET https://api.nuget.org:443/v3/registration5-gz-semver2/scrutor/index.json
2024-10-04T13:14:54.5944268Z   proxy | 2024/10/04 13:14:54 [038] 200 https://api.nuget.org:443/v3/registration5-gz-semver2/scrutor/index.json
2024-10-04T13:14:54.6486890Z updater | 2024/10/04 13:14:54 INFO <job_895966952> Filtered out 3 pre-release versions
2024-10-04T13:14:54.7624443Z   proxy | 2024/10/04 13:14:54 [042] GET https://api.nuget.org:443/v3-flatcontainer/scrutor/5.0.0/scrutor.nuspec
2024-10-04T13:14:54.7840998Z   proxy | 2024/10/04 13:14:54 [042] 200 https://api.nuget.org:443/v3-flatcontainer/scrutor/5.0.0/scrutor.nuspec
2024-10-04T13:14:54.8017479Z updater | 2024/10/04 13:14:54 INFO <job_895966952> Latest version is 5.0.1
2024-10-04T13:14:54.8082735Z updater | 2024/10/04 13:14:54 INFO <job_895966952> Requirements to unlock all
2024-10-04T13:14:54.8093206Z updater | 2024/10/04 13:14:54 INFO <job_895966952> Requirements update strategy 
2024-10-04T13:14:54.8108086Z updater | Finding updated dependencies for Scrutor.
2024-10-04T13:14:54.9253821Z   proxy | 2024/10/04 13:14:54 [044] GET https://api.nuget.org:443/v3-flatcontainer/scrutor/5.0.1/scrutor.nuspec
[-------redacted for brevity ------------------ ]
2024-10-04T13:15:02.8399100Z updater | 2024/10/04 13:15:02 INFO <job_895966952> Updating Scrutor from 5.0.0 to 5.0.1
2024-10-04T13:15:02.8531069Z updater | running NuGet updater:
2024-10-04T13:15:02.8535983Z /opt/nuget/NuGetUpdater/NuGetUpdater.Cli update --repo-root /home/dependabot/dependabot-updater/repo --solution-or-project /home/dependabot/dependabot-updater/repo/src/TestProject.csproj --dependency Scrutor --new-version 5.0.1 --previous-version 5.0.0 --result-output-path /tmp/update-result.json --verbose
2024-10-04T13:15:05.9388843Z   proxy | 2024/10/04 13:15:05 [162] GET https://api.nuget.org:443/v3-flatcontainer/scrutor/5.0.1/scrutor.5.0.1.nupkg
2024-10-04T13:15:05.9629158Z   proxy | 2024/10/04 13:15:05 [162] 200 https://api.nuget.org:443/v3-flatcontainer/scrutor/5.0.1/scrutor.5.0.1.nupkg
2024-10-04T13:15:07.9865439Z   proxy | 2024/10/04 13:15:07 [164] GET https://api.nuget.org:443/v3/registration5-gz-semver2/scrutor/index.json
2024-10-04T13:15:08.0130536Z   proxy | 2024/10/04 13:15:08 [164] 200 https://api.nuget.org:443/v3/registration5-gz-semver2/scrutor/index.json
2024-10-04T13:15:15.1908610Z updater |   No dotnet-tools.json file found.
2024-10-04T13:15:15.1914734Z   No global.json file found.
2024-10-04T13:15:15.1916188Z Running for project file [src/TestProject.csproj]
2024-10-04T13:15:15.1917597Z Updating project [/home/dependabot/dependabot-updater/repo/src/TestProject.csproj]
2024-10-04T13:15:15.1918928Z   Running for SDK-style project
2024-10-04T13:15:15.1920181Z     Found incorrect [PackageVersion] version attribute in [src/Directory.Packages.props].
2024-10-04T13:15:15.1921440Z     Saved [src/Directory.Packages.props].
2024-10-04T13:15:15.1922265Z     Updating lock file
2024-10-04T13:15:15.1923014Z Update complete.
2024-10-04T13:15:15.1923880Z   Writing update result to [/tmp/update-result.json].
2024-10-04T13:15:15.1925085Z updater | 2024/10/04 13:15:15 INFO <job_895966952> update result: {
2024-10-04T13:15:15.1926013Z   "ErrorType": null,
2024-10-04T13:15:15.1926578Z   "ErrorDetails": null
2024-10-04T13:15:15.1927054Z }
2024-10-04T13:15:15.1932429Z updater | The contents of file [src/Directory.Packages.props] were updated.
2024-10-04T13:15:15.1933757Z updater | The contents of file [src/packages.lock.json] were updated.
2024-10-04T13:15:15.1936225Z updater | 2024/10/04 13:15:15 INFO <job_895966952> Submitting Scrutor pull request for creation
2024-10-04T13:15:15.4755120Z   proxy | 2024/10/04 13:15:15 [166] GET https://api.github.com:443/repos/name-redacted/dependabot-bug-repro/commits?per_page=100
2024-10-04T13:15:15.4757050Z 2024/10/04 13:15:15 [166] * authenticating github api request with token for api.github.com
2024-10-04T13:15:15.5940321Z   proxy | 2024/10/04 13:15:15 [166] 200 https://api.github.com:443/repos/name-redacted/dependabot-bug-repro/commits?per_page=100
2024-10-04T13:15:15.7211218Z   proxy | 2024/10/04 13:15:15 [168] GET https://api.nuget.org:443/v3-flatcontainer/scrutor/5.0.1/scrutor.nuspec
2024-10-04T13:15:15.7217133Z   proxy | 2024/10/04 13:15:15 [168] 200 https://api.nuget.org:443/v3-flatcontainer/scrutor/5.0.1/scrutor.nuspec
2024-10-04T13:15:15.7753194Z   proxy | 2024/10/04 13:15:15 [170] GET https://api.github.com:443/repos/khellang/Scrutor/releases?per_page=100
2024-10-04T13:15:15.7754956Z   proxy | 2024/10/04 13:15:15 [170] * authenticating github api request with token for api.github.com
2024-10-04T13:15:15.9032736Z   proxy | 2024/10/04 13:15:15 [170] 200 https://api.github.com:443/repos/khellang/Scrutor/releases?per_page=100
2024-10-04T13:15:15.9547530Z   proxy | 2024/10/04 13:15:15 [172] GET https://api.github.com:443/repos/khellang/Scrutor/contents/
2024-10-04T13:15:15.9549053Z 2024/10/04 13:15:15 [172] * authenticating github api request with token for api.github.com
2024-10-04T13:15:16.0285617Z   proxy | 2024/10/04 13:15:16 [172] 200 https://api.github.com:443/repos/khellang/Scrutor/contents/
2024-10-04T13:15:16.1324718Z   proxy | 2024/10/04 13:15:16 [174] GET https://github.com:443/khellang/Scrutor.git/info/refs?service=git-upload-pack
2024-10-04T13:15:16.1326096Z 2024/10/04 13:15:16 [174] * authenticating git server request (host: github.com)
2024-10-04T13:15:16.1833621Z   proxy | 2024/10/04 13:15:16 [174] 200 https://github.com:443/khellang/Scrutor.git/info/refs?service=git-upload-pack
2024-10-04T13:15:16.2046752Z   proxy | 2024/10/04 13:15:16 [176] GET https://api.github.com:443/repos/khellang/Scrutor/contents/?ref=v5.0.1
2024-10-04T13:15:16.2047993Z 2024/10/04 13:15:16 [176] * authenticating github api request with token for api.github.com
2024-10-04T13:15:16.2991904Z   proxy | 2024/10/04 13:15:16 [176] 200 https://api.github.com:443/repos/khellang/Scrutor/contents/?ref=v5.0.1
2024-10-04T13:15:16.4029311Z   proxy | 2024/10/04 13:15:16 [178] GET https://github.com:443/khellang/Scrutor.git/info/refs?service=git-upload-pack
2024-10-04T13:15:16.4033294Z   proxy | 2024/10/04 13:15:16 [178] 200 https://github.com:443/khellang/Scrutor.git/info/refs?service=git-upload-pack
2024-10-04T13:15:16.5141640Z   proxy | 2024/10/04 13:15:16 [180] GET https://api.github.com:443/repos/khellang/Scrutor/commits?sha=v5.0.0
2024-10-04T13:15:16.5142847Z 2024/10/04 13:15:16 [180] * authenticating github api request with token for api.github.com
2024-10-04T13:15:16.6365317Z   proxy | 2024/10/04 13:15:16 [180] 200 https://api.github.com:443/repos/khellang/Scrutor/commits?sha=v5.0.0
2024-10-04T13:15:16.6652113Z   proxy | 2024/10/04 13:15:16 [182] GET https://api.github.com:443/repos/khellang/Scrutor/commits?sha=v5.0.1
2024-10-04T13:15:16.6654218Z   proxy | 2024/10/04 13:15:16 [182] * authenticating github api request with token for api.github.com
2024-10-04T13:15:16.8142406Z   proxy | 2024/10/04 13:15:16 [182] 200 https://api.github.com:443/repos/khellang/Scrutor/commits?sha=v5.0.1
2024-10-04T13:15:16.8387488Z   proxy | 2024/10/04 13:15:16 [184] GET https://api.github.com:443/repos/khellang/Scrutor/commits?sha=v5.0.0
2024-10-04T13:15:16.8389294Z   proxy | 2024/10/04 13:15:16 [184] 200 https://api.github.com:443/repos/khellang/Scrutor/commits?sha=v5.0.0
2024-10-04T13:15:16.8694932Z   proxy | 2024/10/04 13:15:16 [186] GET https://api.github.com:443/repos/khellang/Scrutor/commits?sha=v5.0.1
2024-10-04T13:15:16.8697403Z 2024/10/04 13:15:16 [186] 200 https://api.github.com:443/repos/khellang/Scrutor/commits?sha=v5.0.1
2024-10-04T13:15:16.8958640Z   proxy | 2024/10/04 13:15:16 [188] GET https://api.github.com:443/repos/khellang/Scrutor/commits?sha=v5.0.0
2024-10-04T13:15:16.8961697Z   proxy | 2024/10/04 13:15:16 [188] 200 https://api.github.com:443/repos/khellang/Scrutor/commits?sha=v5.0.0
2024-10-04T13:15:16.9239638Z   proxy | 2024/10/04 13:15:16 [190] GET https://api.github.com:443/repos/khellang/Scrutor/commits?sha=v5.0.1
2024-10-04T13:15:16.9246017Z   proxy | 2024/10/04 13:15:16 [190] 200 https://api.github.com:443/repos/khellang/Scrutor/commits?sha=v5.0.1
2024-10-04T13:15:16.9698762Z   proxy | 2024/10/04 13:15:16 [192] POST /update_jobs/895966952/create_pull_request
2024-10-04T13:15:17.0196133Z   proxy | 2024/10/04 13:15:17 [192] 400 /update_jobs/895966952/create_pull_request
2024-10-04T13:15:17.0285752Z   proxy | 2024/10/04 13:15:17 [194] POST /update_jobs/895966952/record_update_job_unknown_error
2024-10-04T13:15:17.1055704Z   proxy | 2024/10/04 13:15:17 [194] 204 /update_jobs/895966952/record_update_job_unknown_error
2024-10-04T13:15:17.1522865Z   proxy | 2024/10/04 13:15:17 [196] POST /update_jobs/895966952/record_update_job_error
2024-10-04T13:15:17.1985391Z   proxy | 2024/10/04 13:15:17 [196] 204 /update_jobs/895966952/record_update_job_error
2024-10-04T13:15:17.2442765Z   proxy | 2024/10/04 13:15:17 [198] POST /update_jobs/895966952/increment_metric
2024-10-04T13:15:17.2838730Z   proxy | 2024/10/04 13:15:17 [198] 204 /update_jobs/895966952/increment_metric
2024-10-04T13:15:17.2900562Z   proxy | 2024/10/04 13:15:17 [200] POST /update_jobs/895966952/record_update_job_unknown_error
2024-10-04T13:15:17.3711672Z   proxy | 2024/10/04 13:15:17 [200] 204 /update_jobs/895966952/record_update_job_unknown_error
2024-10-04T13:15:17.3745907Z updater | 2024/10/04 13:15:17 ERROR <job_895966952> Error processing Scrutor (Dependabot::ApiError)
2024-10-04T13:15:17.3747936Z 2024/10/04 13:15:17 ERROR <job_895966952> {"errors":[{"status":400,"title":"Bad Request","detail":"The request contains invalid or unauthorized changes"}]}
2024-10-04T13:15:17.3750952Z 2024/10/04 13:15:17 ERROR <job_895966952> /home/dependabot/dependabot-updater/lib/dependabot/api_client.rb:42:in `block in create_pull_request'
2024-10-04T13:15:17.3755278Z 2024/10/04 13:15:17 ERROR <job_895966952> /home/dependabot/dependabot-updater/vendor/ruby/3.3.0/gems/opentelemetry-api-1.4.0/lib/opentelemetry/trace/tracer.rb:37:in `block in in_span'
2024-10-04T13:15:17.3758905Z 2024/10/04 13:15:17 ERROR <job_895966952> /home/dependabot/dependabot-updater/vendor/ruby/3.3.0/gems/opentelemetry-api-1.4.0/lib/opentelemetry/trace.rb:70:in `block in with_span'
2024-10-04T13:15:17.3762238Z 2024/10/04 13:15:17 ERROR <job_895966952> /home/dependabot/dependabot-updater/vendor/ruby/3.3.0/gems/opentelemetry-api-1.4.0/lib/opentelemetry/context.rb:87:in `with_value'
2024-10-04T13:15:17.3779739Z 2024/10/04 13:15:17 ERROR <job_895966952> /home/dependabot/dependabot-updater/vendor/ruby/3.3.0/gems/opentelemetry-api-1.4.0/lib/opentelemetry/trace.rb:70:in `with_span'
2024-10-04T13:15:17.3785444Z 2024/10/04 13:15:17 ERROR <job_895966952> /home/dependabot/dependabot-updater/vendor/ruby/3.3.0/gems/opentelemetry-api-1.4.0/lib/opentelemetry/trace/tracer.rb:37:in `in_span'
2024-10-04T13:15:17.3788156Z 2024/10/04 13:15:17 ERROR <job_895966952> /home/dependabot/dependabot-updater/lib/dependabot/api_client.rb:34:in `create_pull_request'
2024-10-04T13:15:17.3791168Z 2024/10/04 13:15:17 ERROR <job_895966952> /home/dependabot/dependabot-updater/vendor/ruby/3.3.0/gems/sorbet-runtime-0.5.11444/lib/types/private/methods/call_validation.rb:270:in `bind_call'
2024-10-04T13:15:17.3796036Z 2024/10/04 13:15:17 ERROR <job_895966952> /home/dependabot/dependabot-updater/vendor/ruby/3.3.0/gems/sorbet-runtime-0.5.11444/lib/types/private/methods/call_validation.rb:270:in `validate_call'
2024-10-04T13:15:17.3799468Z updater | 2024/10/04 13:15:17 ERROR <job_895966952> /home/dependabot/dependabot-updater/vendor/ruby/3.3.0/gems/sorbet-runtime-0.5.11444/lib/types/private/methods/_methods.rb:277:in `block in _on_method_added'
2024-10-04T13:15:17.3802430Z 2024/10/04 13:15:17 ERROR <job_895966952> /home/dependabot/dependabot-updater/lib/dependabot/service.rb:57:in `create_pull_request'
2024-10-04T13:15:17.3805448Z 2024/10/04 13:15:17 ERROR <job_895966952> /home/dependabot/dependabot-updater/vendor/ruby/3.3.0/gems/sorbet-runtime-0.5.11444/lib/types/private/methods/call_validation.rb:270:in `bind_call'
2024-10-04T13:15:17.3809188Z updater | 2024/10/04 13:15:17 ERROR <job_895966952> /home/dependabot/dependabot-updater/vendor/ruby/3.3.0/gems/sorbet-runtime-0.5.11444/lib/types/private/methods/call_validation.rb:270:in `validate_call'
2024-10-04T13:15:17.3812926Z updater | 2024/10/04 13:15:17 ERROR <job_895966952> /home/dependabot/dependabot-updater/vendor/ruby/3.3.0/gems/sorbet-runtime-0.5.11444/lib/types/private/methods/_methods.rb:277:in `block in _on_method_added'
2024-10-04T13:15:17.3816847Z updater | 2024/10/04 13:15:17 ERROR <job_895966952> /home/dependabot/dependabot-updater/lib/dependabot/updater/operations/update_all_versions.rb:325:in `create_pull_request'
2024-10-04T13:15:17.3820339Z updater | 2024/10/04 13:15:17 ERROR <job_895966952> /home/dependabot/dependabot-updater/vendor/ruby/3.3.0/gems/sorbet-runtime-0.5.11444/lib/types/private/methods/call_validation.rb:270:in `bind_call'
2024-10-04T13:15:17.3823799Z 2024/10/04 13:15:17 ERROR <job_895966952> /home/dependabot/dependabot-updater/vendor/ruby/3.3.0/gems/sorbet-runtime-0.5.11444/lib/types/private/methods/call_validation.rb:270:in `validate_call'
2024-10-04T13:15:17.3828007Z 2024/10/04 13:15:17 ERROR <job_895966952> /home/dependabot/dependabot-updater/vendor/ruby/3.3.0/gems/sorbet-runtime-0.5.11444/lib/types/private/methods/_methods.rb:277:in `block in _on_method_added'
2024-10-04T13:15:17.3831472Z 2024/10/04 13:15:17 ERROR <job_895966952> /home/dependabot/dependabot-updater/lib/dependabot/updater/operations/update_all_versions.rb:186:in `check_and_create_pull_request'
2024-10-04T13:15:17.3835000Z 2024/10/04 13:15:17 ERROR <job_895966952> /home/dependabot/dependabot-updater/vendor/ruby/3.3.0/gems/sorbet-runtime-0.5.11444/lib/types/private/methods/call_validation.rb:270:in `bind_call'
2024-10-04T13:15:17.3838507Z 2024/10/04 13:15:17 ERROR <job_895966952> /home/dependabot/dependabot-updater/vendor/ruby/3.3.0/gems/sorbet-runtime-0.5.11444/lib/types/private/methods/call_validation.rb:270:in `validate_call'
2024-10-04T13:15:17.3841958Z 2024/10/04 13:15:17 ERROR <job_895966952> /home/dependabot/dependabot-updater/vendor/ruby/3.3.0/gems/sorbet-runtime-0.5.11444/lib/types/private/methods/_methods.rb:277:in `block in _on_method_added'
2024-10-04T13:15:17.3845763Z 2024/10/04 13:15:17 ERROR <job_895966952> /home/dependabot/dependabot-updater/lib/dependabot/updater/operations/update_all_versions.rb:94:in `check_and_create_pr_with_error_handling'
2024-10-04T13:15:17.3849214Z 2024/10/04 13:15:17 ERROR <job_895966952> /home/dependabot/dependabot-updater/vendor/ruby/3.3.0/gems/sorbet-runtime-0.5.11444/lib/types/private/methods/call_validation.rb:270:in `bind_call'
2024-10-04T13:15:17.3852997Z 2024/10/04 13:15:17 ERROR <job_895966952> /home/dependabot/dependabot-updater/vendor/ruby/3.3.0/gems/sorbet-runtime-0.5.11444/lib/types/private/methods/call_validation.rb:270:in `validate_call'
2024-10-04T13:15:17.3856869Z 2024/10/04 13:15:17 ERROR <job_895966952> /home/dependabot/dependabot-updater/vendor/ruby/3.3.0/gems/sorbet-runtime-0.5.11444/lib/types/private/methods/_methods.rb:277:in `block in _on_method_added'
2024-10-04T13:15:17.3860365Z 2024/10/04 13:15:17 ERROR <job_895966952> /home/dependabot/dependabot-updater/lib/dependabot/updater/operations/update_all_versions.rb:59:in `block in perform'
2024-10-04T13:15:17.3863217Z 2024/10/04 13:15:17 ERROR <job_895966952> /home/dependabot/dependabot-updater/lib/dependabot/updater/operations/update_all_versions.rb:59:in `each'
2024-10-04T13:15:17.3866393Z 2024/10/04 13:15:17 ERROR <job_895966952> /home/dependabot/dependabot-updater/lib/dependabot/updater/operations/update_all_versions.rb:59:in `perform'
2024-10-04T13:15:17.3869600Z updater | 2024/10/04 13:15:17 ERROR <job_895966952> /home/dependabot/dependabot-updater/vendor/ruby/3.3.0/gems/sorbet-runtime-0.5.11444/lib/types/private/methods/call_validation.rb:270:in `bind_call'
2024-10-04T13:15:17.3873138Z updater | 2024/10/04 13:15:17 ERROR <job_895966952> /home/dependabot/dependabot-updater/vendor/ruby/3.3.0/gems/sorbet-runtime-0.5.11444/lib/types/private/methods/call_validation.rb:270:in `validate_call'
2024-10-04T13:15:17.3877152Z 2024/10/04 13:15:17 ERROR <job_895966952> /home/dependabot/dependabot-updater/vendor/ruby/3.3.0/gems/sorbet-runtime-0.5.11444/lib/types/private/methods/_methods.rb:277:in `block in _on_method_added'
2024-10-04T13:15:17.3880691Z 2024/10/04 13:15:17 ERROR <job_895966952> /home/dependabot/dependabot-updater/lib/dependabot/updater/operations/group_update_all_versions.rb:137:in `block in run_ungrouped_dependency_updates'
2024-10-04T13:15:17.3883769Z 2024/10/04 13:15:17 ERROR <job_895966952> /home/dependabot/dependabot-updater/lib/dependabot/updater/operations/group_update_all_versions.rb:122:in `each'
2024-10-04T13:15:17.3887287Z 2024/10/04 13:15:17 ERROR <job_895966952> /home/dependabot/dependabot-updater/lib/dependabot/updater/operations/group_update_all_versions.rb:122:in `run_ungrouped_dependency_updates'
2024-10-04T13:15:17.3890693Z updater | 2024/10/04 13:15:17 ERROR <job_895966952> /home/dependabot/dependabot-updater/vendor/ruby/3.3.0/gems/sorbet-runtime-0.5.11444/lib/types/private/methods/call_validation.rb:270:in `bind_call'
2024-10-04T13:15:17.3895185Z 2024/10/04 13:15:17 ERROR <job_895966952> /home/dependabot/dependabot-updater/vendor/ruby/3.3.0/gems/sorbet-runtime-0.5.11444/lib/types/private/methods/call_validation.rb:270:in `validate_call'
2024-10-04T13:15:17.3898666Z 2024/10/04 13:15:17 ERROR <job_895966952> /home/dependabot/dependabot-updater/vendor/ruby/3.3.0/gems/sorbet-runtime-0.5.11444/lib/types/private/methods/_methods.rb:277:in `block in _on_method_added'
2024-10-04T13:15:17.3901822Z 2024/10/04 13:15:17 ERROR <job_895966952> /home/dependabot/dependabot-updater/lib/dependabot/updater/operations/group_update_all_versions.rb:65:in `perform'
2024-10-04T13:15:17.3905429Z 2024/10/04 13:15:17 ERROR <job_895966952> /home/dependabot/dependabot-updater/vendor/ruby/3.3.0/gems/sorbet-runtime-0.5.11444/lib/types/private/methods/call_validation.rb:270:in `bind_call'
2024-10-04T13:15:17.3908862Z 2024/10/04 13:15:17 ERROR <job_895966952> /home/dependabot/dependabot-updater/vendor/ruby/3.3.0/gems/sorbet-runtime-0.5.11444/lib/types/private/methods/call_validation.rb:270:in `validate_call'
2024-10-04T13:15:17.3912357Z 2024/10/04 13:15:17 ERROR <job_895966952> /home/dependabot/dependabot-updater/vendor/ruby/3.3.0/gems/sorbet-runtime-0.5.11444/lib/types/private/methods/_methods.rb:277:in `block in _on_method_added'
2024-10-04T13:15:17.3915429Z 2024/10/04 13:15:17 ERROR <job_895966952> /home/dependabot/dependabot-updater/lib/dependabot/updater.rb:45:in `run'
2024-10-04T13:15:17.3918001Z 2024/10/04 13:15:17 ERROR <job_895966952> /home/dependabot/dependabot-updater/lib/dependabot/update_files_command.rb:44:in `block in perform_job'
2024-10-04T13:15:17.3921113Z 2024/10/04 13:15:17 ERROR <job_895966952> /home/dependabot/dependabot-updater/vendor/ruby/3.3.0/gems/opentelemetry-api-1.4.0/lib/opentelemetry/trace/tracer.rb:37:in `block in in_span'
2024-10-04T13:15:17.3924504Z 2024/10/04 13:15:17 ERROR <job_895966952> /home/dependabot/dependabot-updater/vendor/ruby/3.3.0/gems/opentelemetry-api-1.4.0/lib/opentelemetry/trace.rb:70:in `block in with_span'
2024-10-04T13:15:17.3927710Z updater | 2024/10/04 13:15:17 ERROR <job_895966952> /home/dependabot/dependabot-updater/vendor/ruby/3.3.0/gems/opentelemetry-api-1.4.0/lib/opentelemetry/context.rb:87:in `with_value'
2024-10-04T13:15:17.3930886Z 2024/10/04 13:15:17 ERROR <job_895966952> /home/dependabot/dependabot-updater/vendor/ruby/3.3.0/gems/opentelemetry-api-1.4.0/lib/opentelemetry/trace.rb:70:in `with_span'
2024-10-04T13:15:17.3934125Z 2024/10/04 13:15:17 ERROR <job_895966952> /home/dependabot/dependabot-updater/vendor/ruby/3.3.0/gems/opentelemetry-api-1.4.0/lib/opentelemetry/trace/tracer.rb:37:in `in_span'
2024-10-04T13:15:17.3936798Z 2024/10/04 13:15:17 ERROR <job_895966952> /home/dependabot/dependabot-updater/lib/dependabot/update_files_command.rb:18:in `perform_job'
2024-10-04T13:15:17.3939201Z 2024/10/04 13:15:17 ERROR <job_895966952> /home/dependabot/dependabot-updater/lib/dependabot/base_command.rb:37:in `run'
2024-10-04T13:15:17.3940810Z 2024/10/04 13:15:17 ERROR <job_895966952> bin/update_files.rb:46:in `<main>'
2024-10-04T13:15:17.3942201Z updater | 2024/10/04 13:15:17 INFO <job_895966952> Checking if xunit 2.9.1 needs updating
2024-10-04T13:15:17.4994711Z   proxy | 2024/10/04 13:15:17 [202] GET https://api.nuget.org:443/v3/registration5-gz-semver2/xunit/index.json
2024-10-04T13:15:17.5389833Z   proxy | 2024/10/04 13:15:17 [202] 200 https://api.nuget.org:443/v3/registration5-gz-semver2/xunit/index.json
2024-10-04T13:15:17.5680818Z updater | 2024/10/04 13:15:17 INFO <job_895966952> Filtered out 74 pre-release versions
2024-10-04T13:15:17.6803146Z   proxy | 2024/10/04 13:15:17 [208] GET https://api.nuget.org:443/v3-flatcontainer/xunit/2.9.1/xunit.nuspec
2024-10-04T13:15:17.7025556Z   proxy | 2024/10/04 13:15:17 [208] 200 https://api.nuget.org:443/v3-flatcontainer/xunit/2.9.1/xunit.nuspec
2024-10-04T13:15:17.8215218Z   proxy | 2024/10/04 13:15:17 [210] GET https://api.nuget.org:443/v3-flatcontainer/xunit/2.9.1/xunit.2.9.1.nupkg
2024-10-04T13:15:17.8460065Z   proxy | 2024/10/04 13:15:17 [210] 200 https://api.nuget.org:443/v3-flatcontainer/xunit/2.9.1/xunit.2.9.1.nupkg
2024-10-04T13:15:17.9524824Z   proxy | 2024/10/04 13:15:17 [212] GET https://api.nuget.org:443/v3-flatcontainer/xunit/2.9.2/xunit.nuspec
2024-10-04T13:15:17.9746514Z   proxy | 2024/10/04 13:15:17 [212] 200 https://api.nuget.org:443/v3-flatcontainer/xunit/2.9.2/xunit.nuspec
2024-10-04T13:15:18.0845085Z   proxy | 2024/10/04 13:15:18 [214] GET https://api.nuget.org:443/v3-flatcontainer/xunit/2.9.2/xunit.2.9.2.nupkg
2024-10-04T13:15:18.1391719Z   proxy | 2024/10/04 13:15:18 [214] 200 https://api.nuget.org:443/v3-flatcontainer/xunit/2.9.2/xunit.2.9.2.nupkg
2024-10-04T13:15:18.1463138Z updater | 2024/10/04 13:15:18 INFO <job_895966952> Latest version is 2.9.2
2024-10-04T13:15:18.1473118Z updater | 2024/10/04 13:15:18 INFO <job_895966952> Requirements to unlock all
2024-10-04T13:15:18.1479187Z 2024/10/04 13:15:18 INFO <job_895966952> Requirements update strategy 
2024-10-04T13:15:18.1480353Z updater | Finding updated dependencies for xunit.
2024-10-04T13:15:18.2501624Z   proxy | 2024/10/04 13:15:18 [216] GET https://api.nuget.org:443/v3-flatcontainer/xunit/2.9.2/xunit.nuspec
2024-10-04T13:15:18.2505833Z   proxy | 2024/10/04 13:15:18 [216] 200 https://api.nuget.org:443/v3-flatcontainer/xunit/2.9.2/xunit.nuspec
2024-10-04T13:15:18.2559846Z updater | 2024/10/04 13:15:18 INFO <job_895966952> Updating xunit from 2.9.1 to 2.9.2
2024-10-04T13:15:18.3183807Z   proxy | 2024/10/04 13:15:18 [218] POST /update_jobs/895966952/record_update_job_error
2024-10-04T13:15:18.3743034Z   proxy | 2024/10/04 13:15:18 [218] 204 /update_jobs/895966952/record_update_job_error
2024-10-04T13:15:18.3753231Z updater | 2024/10/04 13:15:18 INFO <job_895966952> Handled error whilst updating xunit: update_not_possible {:dependencies=>["xunit"]}
2024-10-04T13:15:18.4212622Z   proxy | 2024/10/04 13:15:18 [220] PATCH /update_jobs/895966952/mark_as_processed
2024-10-04T13:15:18.4939078Z   proxy | 2024/10/04 13:15:18 [220] 204 /update_jobs/895966952/mark_as_processed
2024-10-04T13:15:18.4944919Z updater | 2024/10/04 13:15:18 INFO <job_895966952> Finished job processing
2024-10-04T13:15:18.4965117Z updater | 2024/10/04 13:15:18 INFO Results:
2024-10-04T13:15:18.4972887Z Dependabot encountered '2' error(s) during execution, please check the logs for more details.
2024-10-04T13:15:18.4974210Z +-------------------------------+
2024-10-04T13:15:18.4974969Z | Dependencies failed to update |
2024-10-04T13:15:18.4975631Z +---------+---------------------+
2024-10-04T13:15:18.4976250Z | Scrutor | unknown_error       |
2024-10-04T13:15:18.4976886Z | xunit   | update_not_possible |
2024-10-04T13:15:18.4977603Z +---------+---------------------+
2024-10-04T13:15:18.6731606Z Failure running container d3fcc3070906824ff3832801c49309ab362c4fb1ed649365b578c307611c1c6c
2024-10-04T13:15:18.6999718Z Cleaned up container d3fcc3070906824ff3832801c49309ab362c4fb1ed649365b578c307611c1c6c
2024-10-04T13:15:18.7139790Z   proxy | 2024/10/04 13:15:18 7/107 calls cached (6%)
2024-10-04T13:15:18.7142865Z 2024/10/04 13:15:18 Posting metrics to remote API endpoint
2024-10-04T13:15:19.0502338Z ##[error]Dependabot encountered an error performing the update

Error: The updater encountered one or more errors.

For more information see: https://github.com/name-redacted/dependabot-bug-repro/network/updates/895966952 (write access to the repository is required to view the log)
2024-10-04T13:15:19.0516257Z 🤖 ~ finished: error reported to Dependabot ~
2024-10-04T13:15:19.0602440Z Post job cleanup.
2024-10-04T13:15:19.2151981Z Cleaning up orphan processes
```
</details>

### Smallest manifest that reproduces the issue

_No response_

--- a/nuget/lib/dependabot/nuget/file_updater.rb
+++ b/nuget/lib/dependabot/nuget/file_updater.rb
@@ -20,6 +20,7 @@ module Dependabot
       def self.updated_files_regex
         [
           /.*\.([a-z]{2})?proj$/, # Matches files with any extension like .csproj, .vbproj, etc., in any directory
+          /packages\.lock\.json/,         # Matches packages.lock.json in any directory
           /packages\.config$/i,           # Matches packages.config in any directory
           /app\.config$/i,                # Matches app.config in any directory
           /web\.config$/i,                # Matches web.config in any directory

--- a/nuget/spec/dependabot/nuget/file_updater_spec.rb
+++ b/nuget/spec/dependabot/nuget/file_updater_spec.rb
@@ -123,6 +123,7 @@ RSpec.describe Dependabot::Nuget::FileUpdater do
           "project.csproj",
           "library.fsproj",
           "app.vbproj",
+          "packages.lock.json",
           "packages.config",
           "app.config",
           "web.config",

